resource/css/skosmos.css

```diff
@@ -860,6 +860,15 @@ body {
     border-radius: 0;
   }
 
+ /* About page
+   *****************************************/
+  #about-content a {
+    font-size: 0,8ems;
```

Comment:
This uses the wrong decimal point (comma) and wrong unit (ems). It should be:

    font-size: 0.8em;

Reply:
Fixed